R/example.R

```diff
@@ -399,29 +399,34 @@ example_notebook_alternating_function_in_files <- function(folder_path="/tmp", f
 #' @return the JSON path of the newly created issue issue tracker
 #' @export
 #' @keywords internal
-example_jira_issue_components <- function(folder_path="/tmp",folder_name) {
+example_jira_issue_components <- function(folder_path, folder_name) {
 
   # Create folder & repo
-  folder_path <- io_make_folder(folder_path=folder_path, folder_name = folder_name)
-
-  issue1 <- make_jira_issue(jira_domain_url = "https://project.org/jira",
-                            issue_key = "GERONIMO-123",
-                            issue_type = "New Feature",
-                            status = "Open",
-                            resolution = "Finished",
-                            title = "This is a summary.",
-                            description = "The new features have been implemented.",
-                            components = "x-core;x-spring",
-                            creator_name = "Bob",
-                            reporter_name = "Joe",
-                            assignee_name = "Moe",
+  folder_path <- io_make_folder(folder_path = folder_path, folder_name = folder_name)
+
+  issue1 <- make_jira_issue(
+    jira_domain_url = "https://project.org/jira",
+    issue_key = "GERONIMO-123",
+    project_key = "GERONIMO",
+    summary = "Summary of new feature",
+    description = "The new features have been implemented",
+    issue_type = "New Feature",
+    resolution = "Finished",
+    priority = "Minor",
+    status = "Open",
+    labels = c("pull-request-available"),
+    components = c("jira", "mail"),
+    affects_versions = c("3.4.3"),
+    fix_versions = c("3.4.2"),
+    assignee_name = "Moe",
+    creator_name = "Bob",
+    reporter_name = "Joe"
   )
 
-  # issues <- c(list(issue1))
   issues <- list(issue1)
 
-  jira_json_path <- make_jira_issue_tracker(issues,
-                                            save_filepath=file.path(folder_path,"issue_two_components.json"))
+  jira_json_path <- make_jira_issue_tracker(
+    issues, save_filepath=file.path(folder_path, "GERONIMO_issues_1121646814_1121719175.json"))
```

Comment:
You should not identify any particular project in the examples, and the file names should reflect the intent of the example.